src/components/implementation/tests/micro_xcores/Makefile

```diff
@@ -0,0 +1,10 @@
+C_OBJS=micro_xcores.o test_ipi_n_n.o test_ipi_interference.o test_ipi_switch.o test_ipi_roundtrip.o
```

Comment:
Do you have the cross-core stuff booting on qemu/emulation?  I know the #s and values are only relevant on the real hardware, but does it boot without error on multicore?

Reply:
honestly, I don't know since I can't nest virtualize. I think @phanikishoreg took a look but I might be worng